pkg/log/logshipper/logshipper.go

```diff
@@ -192,42 +173,142 @@ func filterResults(keyvals ...interface{}) {
 	}
 }
 
-// addDeviceIdentifyingAttributesToLogger gets device identifiers from the server-provided
+// updateDevideIdentifyingAttributes gets device identifiers from the server-provided
 // data and adds them as attributes on the logger.
-func (ls *LogShipper) addDeviceIdentifyingAttributesToLogger() {
-	additionalSlogAttrs := make([]slog.Attr, 0)
+func (ls *LogShipper) updateDevideIdentifyingAttributes() error {
+	deviceInfo := make(map[string]string)
 
 	versionInfo := version.Version()
+	deviceInfo["version"] = versionInfo.Version
 	ls.shippingLogger = log.With(ls.shippingLogger, "version", versionInfo.Version)
-	additionalSlogAttrs = append(additionalSlogAttrs, slog.String("version", versionInfo.Version))
 
-	if deviceId, err := ls.knapsack.ServerProvidedDataStore().Get([]byte("device_id")); err != nil {
-		level.Debug(ls.baseLogger).Log("msg", "could not get device id", "err", err)
-	} else {
-		ls.shippingLogger = log.With(ls.shippingLogger, "k2_device_id", string(deviceId))
-		additionalSlogAttrs = append(additionalSlogAttrs, slog.String("k2_device_id", string(deviceId)))
+	for _, key := range []string{"device_id", "munemo", "organization_id", "serial_number"} {
```

Comment:
this drops the `k2` prefix from some of the attributes, is that okay?

Reply:
I don't feel strongly. 

I often like `k2` because it makes it very clear what I'm looking at. But I'm not sure we need that disambiguation